### DISCUSSION TITLE
Way of sharing / importing brews

### DISCUSSION THREAD

**Discussion by vincentvdwal** [graphefruit/Beanconqueror#812] [Ideas]
Hey there.

I was wondering. Since the link sharing works so well with beans. Are there any plans to create some similar way of sharing brews, so one could generate a link with a brew and there after import it in the app. This would be genius for some Devices with PID, so brews can be automatically generated.

Kind regards,

Vincent

**Comment by graphefruit** [comment]
Hello @vincvdwal ,
thanks for the ticket.
This is sadly not possible.

Bean, Grinder, Preparation method, have all an unique id.
A brew is a combination of a bean, grinder and preparation method all those information are not existing on the other user side.

Maybe we'll get the possibility with a "sharing recipe" but this will be not default to a grinder, preparation method or something else.
